waiter/integration/waiter/metrics_output_test.clj

```diff
@@ -140,3 +140,63 @@
         (is (>= (get-in aggregate-metrics ["counters" "request-counts" "total"]) num-requests)))
 
       (delete-service waiter-url service-id))))
+
+
+(defmacro get-p-value
```

Comment:
I'd prefer `get-percentile-value` as I wasn't sure what `p` referred to here initally.

Reply:
Done.